### PR TITLE
Show assignee name in success banner

### LINKED ISSUE #14217: Case Movement_Non-Blocking Tasks | Assign/Reassign Case to Attorney Banner Inconsistency
## Description
The success banner when a case is assigned/reassigned to an attorney should display the same copy as assigning/reassigning a case to a judge

## Acceptance criteria
- [ ] Attorney assign/reassigning displays the following copy in the success banner "Task Assigned To {User} 

## Background/context/resources
Located during CM Dogfooding
Check the [dogfooding google doc](https://docs.google.com/document/d/1JbCt-TYW3wmLDpFbLoxFxMeX8YAlUauq3viaJm_A_J4/edit?usp=sharing) for screenshots
## Screenshots
image

### PR DESCRIPTION
**Pull request by hschallhorn**
Resolves #14217

### Description
Adds assignee name to success banner when assigning to an attorney. Also displays if this is an initial assignment or a reassignment

### Acceptance Criteria
- [ ] When assigning a case to an attorney, the assignee's name appears in the success banner
- [ ] When reassigning a case to an attorney, the assignee's name appears in the success banner and displays that this was a reassignment

### Testing Plan
1. Log in as BVAAABSHIRE and go to any appeal with a judge assign task
1. Assign the case to an attorney
1. Confirm atty name is in banner
1. Go back to case details and reassign the case to another attorney
1. Confirm "Reassigned" and attorney name appear in the banner
1. Log in as BVARDUNKLE
1. Go to any case that a judge has an assign task on
1. Assign the case to someone from the "other" dropdown
1. Confirm atty's name is in the success banner

### User Facing Changes

 BEFORE|REASSIGN|ASSIGN
 ---|---|---
81324824-83780900-9065-11ea-9c58-e878d8bf7115|Screen Shot 2020-06-01 at 3 26 42 PM|Screen Shot 2020-06-01 at 3 27 00 PM

### CODE REVIEW

**Comment by yoomlam** [comment]
May want another ticket for this: If I assign the case from the Case Details page (coming from the Case Assignment page), the subsequent page shows the correct banner but it is the "Your cases" page (rather than the Case Assignment page). Also the banner and "Switch views" button is overlapping.  
image

**Comment by hschallhorn** [comment]
#12662 !

**Comment by yoomlam** [comment]
Unsuccessful reassign? No error banner and case is still assigned to Shields after doing this:
unsuccessful-reassign

**Comment by hschallhorn** [comment]
#14437!

**Comment by yoomlam** [comment]
Do we want to distinguish users who happen to have the same exact name (for example by showing their CSS_ID as well)?
image

**Comment by yoomlam** [comment]
Inconsistency: Button refers to "cases" and banner refers to "tasks"
image

File: client/app/queue/components/AssignToAttorneyWidget.jsx [hunk 1]
```diff
@@ -116,17 +131,21 @@ class AssignToAttorneyWidget extends React.PureComponent {
 
     return this.props.onTaskAssignment(
       { tasks: selectedTasks,
-        assigneeId,
+        assigneeId: assignee.id.toString(),
```
Comment (yoomlam) [inline client/app/queue/components/AssignToAttorneyWidget.jsx:134]: Why is the `id` converted to a string? `tasks_controller.rb` treats it as a number.

Comment (hschallhorn) [reply client/app/queue/components/AssignToAttorneyWidget.jsx:134]: Case counts in the judge assign sidebar were not updating without `toString()`. Looks like this is due to the fact that we unnecessarily coerce id to string in  `incrementTaskCountForAttorney` https://github.com/department-of-veterans-affairs/caseflow/blob/96fa3863bd31850bc7deb2b8816f70c3d0cc6926/client/app/queue/reducers.js#L439 Will update! Nice catch!

**Pull request merged**